moveit_core/kinematics_base/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ set(MOVEIT_LIB_NAME moveit_kinematics_base)
 
 add_library(${MOVEIT_LIB_NAME} src/kinematics_base.cpp)
 set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION ${${PROJECT_NAME}_VERSION})
+target_compile_options(${MOVEIT_LIB_NAME} PRIVATE -Wno-deprecated-declarations)
```

Comment:
what is the reasoning for this? can we keep this warning on? if not, add comment as to why not

Reply:
Need to check.

Reply:
Note, that I don't disable deprecation warnings per se, but only for building `kinematics_base.cpp`.
Deprecation of the variables is foremost to warn derived classes, which still use those variables.